engines/glk/zcode/detection_tables.h

```diff
@@ -3907,7 +3907,7 @@ const FrotzGameDescription FROTZ_GAMES[] = {
 	ENTRY0("bryantcollection", "090401", "2c331a7c183ef29ea13b68075a58c73b", 528620),
 	ENTRY0("bse", "970110", "d260f722540f8c24a8c5b88778c76261", 89088),
 	ENTRY0("building", "030706", "cc5ab40754fde9739113ef2524ef66b5", 253440),
-	ENTRY0("bureaucrocy_zcode", "141113", "7dcbbc724b7416ad0b80b3159c7e3525", 387606),
+	ENTRY0("bureaucrocy_zcode", "141113/NotWorking", "7dcbbc724b7416ad0b80b3159c7e3525", 387606),
```

Comment:
This looks a bit weird, if you really want to mark a game as unsupported you could have a look at the `findGame` method in detection.cpp, that marks a few games as `kUnstableGame`, you could to the same for these

Reply:
I've read the code but (imho of course) this approach seems to be practically difficult to follow.

Rather... as in adrifit/detection_tables.h, why not a wrapping like the following: 
```
#ifndef RELEASE_BUILD
    ENTRY0("bureaucrocy_zcode", "141113", "7dcbbc724b7416ad0b80b3159c7e3525", 387606),
#endif
```
At least, I have the same way to mark a game as not supported for each subengine

Reply:
This is easier but the end result is different; if you set the `kUnstableGame` ScummVM shows a warning screen when starting the game, if you use this approach you remove the detection entry completely in release builds

Reply:
- https://github.com/scummvm/scummvm/pull/5388/commits/d054a201ea1d4bb8dffad8e9274f6deb9d1af883
- https://github.com/scummvm/scummvm/pull/5388/commits/aef60a2605d87e7664beda566a8d9c02258b6769

Reply:
@tag2015 regarding this specific comment, is this the right way to use findGame method?

Reply:
> @tag2015 regarding this specific comment, is this the right way to use findGame method?

Yes this looks good, thanks!